lang/en.json

```
@@ -1,159 +1,159 @@
 {
     "TagManager": {
-        "AddThisTagDescription": "Add the AddThis social media buttons and more to your website.",
+        "AddThisTagDescription": "Add the AddThis social-media buttons and more to your website.",
         "AddThisTagPubIdTitle": "AddThis PubId",
-        "AddThisTagPubIdDescription": "Create a new site in Shareaholic and paste the site ID here",
+        "AddThisTagPubIdDescription": "Create a new website in Shareaholic and paste the site ID here",
         "AddThisParentSelectorTitle": "Inline Tools Parent Selector",
         "AddThisParentSelectorDescription": "Optionally enter a CSS selector to the element where the Inline Tools should be added.",
         "BingUETTagIdTitle": "ID",
-        "BingUETTagIdDescription": "You can find the Bing Ad ID when creating a new tracking code in the Bing Ad Campaign Manager.",
+        "BingUETTagIdDescription": "You can find the Microsoft Advertising account number when creating a new tracking code in the Microsoft Advertising Editor tool.",
         "BugsnagTagApiKeyTitle": "API key",
         "BugsnagTagApiKeyDescription": "You can find your API key in your project settings from your Bugsnag dashboard.",
         "BugsnagTagCollectUserIpTitle": "collect user IP",
-        "BugsnagTagCollectUserIpDescription": "This should be disable if you don't want to track personal information about your users.",
+        "BugsnagTagCollectUserIpDescription": "This should be turned off if you don't want to track personal info about your users.",
         "CustomHtmlTagTitle": "Custom HTML",
-        "CustomHtmlTagDescriptionText": "This tag is ideal when you need to add for example custom styles or custom JavaScript or when you are looking for a specific tag which is not yet supported. With this tag you can append any HTML to the bottom of your page, add styles, or execute JavaScript. Note: You can replace content within the HTML with variables by putting a variable name in curly brackets like this {{PageUrl}}.",
+        "CustomHtmlTagDescriptionText": "An ideal tag to add e.g. custom styles and custom JavaScript, or when you are looking for a specific tag not yet supported. With it you can append any HTML to the bottom of your web page, add styles, or execute JavaScript. Note: Content within the HTML can be replaced with variables by putting a variable name in curly brackets like this {{PageUrl}}.",
         "CustomHtmlTagHelpText": "%1$sLearn more%2$s",
         "CustomHtmlHtmlPositionTitle": "Position",
-        "CustomHtmlHtmlPositionDescription": "Define the position of where the HTML should be inserted into your website.",
+        "CustomHtmlHtmlPositionDescription": "Define the position of where to instart the HTML into your website.",
```

```suggestion
        "CustomHtmlHtmlPositionDescription": "Define the position of where to insert the HTML into your website.",
```
Good catch. Was thinking of two things, and the resolved discussion of why it isn't "onto your website".